### PR TITLE
ci: skip coveralls when api not available

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [x] Other... Please describe:

## What is the current behavior?
 
Failed jobs

## What is the new behavior?

Try skip not available api  

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
         "i18n": "nx xi18n --output-path src/locale",
         "*** Workflow ***": "",
         "test": "nx test --browsers=ChromeHeadless --code-coverage=true --",
+        "many": "npm run-script env -- nx run-many --parallel --maxParallel=5",
```
Comment (IKatsuba) [inline package.json:20]: `npm run-script env`
зачем?
`maxParallel=5` кажется, что это too mach для одного агента.

```suggestion
        "many": "nx run-many --parallel --maxParallel=3",
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -41,65 +41,68 @@ jobs:
     - name: Install dependencies
       run: npm ci
     # End of setup
-    - run: npm run test cdk
-    - name: Coveralls
+    - name: Run parallel tests
+      run: |
+          npm run many -- \
```
Comment (IKatsuba) [inline .github/workflows/test.yml:46]: оставь todo, что нужно перенести на affected и собарать покрытие только у заафекченных

File: package.json (hunk 1 above)
Comment (splincode) [reply package.json:20]: what do you think about move this parameter to environments?

`on ci`

```
$ npm run many -- --maxParallel=${{ NX_PARALLEL_COUNT }}
```

or another name for `NX_PARALLEL_COUNT`?

File: .github/workflows/test.yml (hunk 2 above)
Comment (splincode) [reply .github/workflows/test.yml:46]: yes, sure)

File: package.json (hunk 1 above)
Comment (IKatsuba) [reply package.json:20]: Maybe `NX_MAX_PARALLEL`?

**Review by IKatsuba — APPROVED** [review]

**Pull request merged**